docker-compose.yml

```diff
@@ -83,7 +83,10 @@ services:
       - ./docs:/usr/src/code/docs
       - ./public:/usr/src/code/public
       - ./src:/usr/src/code/src
-      - ./dev:/usr/src/code/dev
+      - ./dev:/usr/local/dev
+      - ./vendor/utopia-php/platform:/usr/src/code/vendor/utopia-php/platform
+      - ./vendor/utopia-php/framework:/usr/src/code/vendor/utopia-php/framework
```

Comment:
We've to remove all vendor mounts from the docker compose file

Reply:
fixed